tasks/common.yml

```diff
@@ -63,3 +63,15 @@
   file:
     state: "directory"
     path: "{{ archivematica_src_dir }}"
+
+#
+# Set archivematica_src_ca_custom_bundle if not defined
+#
+
+- set_fact:
+    archivematica_src_ca_custom_bundle: "/etc/ssl/certs/ca-certificates.crt"
+  when: archivematica_src_ca_custom_bundle is undefined and ansible_os_family == "Debian"
+
+- set_fact:
+    archivematica_src_ca_custom_bundle: "/etc/pki/tls/certs/ca-bundle.crt"
+  when: archivematica_src_ca_custom_bundle is undefined and ansible_os_family == "RedHat"
```

Comment:
This makes sense because if you wanted to add them to `vars/{Debian/RedHat}.yml` then we would be making really hard for the user to override them am I right?